packages/react-composites/tests/browser/call/utils.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { encodeQueryData } from '../common/utils';
+import { TestCallingState } from './TestCallingState';
+
+/**
+ * Create the test URL.
+ * @param serverUrl - URL of webpage to test, this is typically https://localhost:3000
+ * @param user - Test user the props of which populate query search parameters
+ * @param state - State
```

Comment:
This `@param` is not very useful. Since this is a test function, just remove it if there's nothing beyond the type you want to say.